### PR TITLE
E2E test for kubectl expose

### PR DESCRIPTION
**Pull request by mwielgus**
As requested in #10764.

### CODE REVIEW

**Comment by mwielgus** [comment]
cc @jlowdermilk

File: test/e2e/kubectl.go [hunk 1]
```diff
@@ -202,6 +202,65 @@ var _ = Describe("Kubectl client", func() {
 		})
 	})
 
+	Describe("Kubectl expose", func() {
+		It("should check if kubectl expose creates services for rc", func() {
```
Comment (jlowdermilk) [inline test/e2e/kubectl.go]: Nit: shorten to "should create services for rc". Describe() already contains `kubectl expose`

**Comment by jlowdermilk** [comment]
lgtm

File: test/e2e/kubectl.go (hunk 1 above)
Comment (mwielgus) [inline test/e2e/kubectl.go]: Shortened.

**Comment by erictune** [comment]
judgement call: adding tests at this point is more risk of flake than chance of catch a bug.  therefore, move to v1-post.  if you did find a bug when writing this test, please call that out and we can decide if test + fix merits v1.

**Comment by wojtek-t** [comment]
@k8s-bot test this please

**Pull request merged**